### PR TITLE
Added more content

### PR DESCRIPTION
**Pull request by BTmathic**
Clarified definition, and gave examples of why primes are important and useful

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/mathematics/prime-numbers/index.md [hunk 1]
```diff
@@ -3,18 +3,14 @@ title: Prime Numbers
...
+For example, there are no '[fast](https://en.wikipedia.org/wiki/General_number_field_sieve)' ways to find prime factors, so while finding the prime factors of 6 above is easy, it is only easy because 6 is small. Finding the prime factors of a number with hundreds or thousands of digits, however, is exceptionally time consuming when there are no [polynomial time](https://en.wikipedia.org/wiki/Time_complexity#Superpolynomial_time) algorithms.
 
+There is no formula for primes, i.e., there is no formula that you can plug any natural number `n` in and get the `n-th` prime from it. The distribution of primes is unknown, i.e., after the `n-th` prime there is no formula for how much larger the `(n+1)-th` prime is. However, it is known that there are [arbitrarily large](https://math.stackexchange.com/a/1675495/12133) gaps between consecutive primes. It is also known that there are infinitely many pairs of consecutive primes that differ by [at most 246](https://en.wikipedia.org/wiki/Twin_prime#History).
 
+You can also win [$1 000 000](https://en.wikipedia.org/wiki/Riemann_hypothesis) for gaining (and demonstrating) new knowledge about the prime numbers.
```
Comment (thecodingaviator) [inline guide/english/mathematics/prime-numbers/index.md]: I don't think it's a really nice idea to add this here. @RandellDawson Any inputs?

Comment (RandellDawson) [reply guide/english/mathematics/prime-numbers/index.md]: I agree this should be removed from the changes.

Comment (thecodingaviator) [inline guide/english/mathematics/prime-numbers/index.md]:
```suggestion

```
Please remove this line @BTmathic

**Comment by BTmathic** [comment]
Ah, sorry. I thought it would be nice to mention that, while simple, there is still an incredible interest in prime numbers. But it has been removed if that's not appropriate.

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**